eigen.spec

```diff
@@ -1,7 +1,7 @@
-### RPM external eigen 011e0db31d1bed8b7f73662be6d57d9f30fa457a
+### RPM external eigen 831133c
 ## INITENV +PATH PKG_CONFIG_PATH %{i}/share/pkgconfig
 ## NOCOMPILER
-%define tag fd0c254c0a9adc47c094257ce6256ab4d94b956e
+%define tag 76141596f2a3f06b1659d00dc4a4dcccd8750a7d
```

Comment:
Did we already try to submit upstream the two changes we apply locally o the Eigen sources ?

Reply:
not as far as I know - I didn't do it and not sure if our changes are generic enough to propose them